### PR TITLE
Added favicon to the blog.

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

**Comment by sidntrivedi** [comment]
@ryzokuken @saranshbarua Please review. 

Tried to use this plugin. https://github.com/Creatiwity/gatsby-plugin-favicon in order to add a favicon.

**Comment by ryzokuken** [comment]
@sidntrivedi012 screenshot please?

**Comment by sidntrivedi** [comment]
@ryzokuken Idk why Gatsby is not setting up on my system. It is giving an error https://paste.fedoraproject.org/paste/fZQa2UaFlNX3tXCofOAiwg. 

So,I couldn't run it. But, having seen the documentation of the favicon plugin, it seemed easy and so I made a PR.

**Comment by sidntrivedi** [comment]
@ryzokuken Not able to debug the error. Please help.

**Comment by sidntrivedi** [comment]
This one was debugged by installing Gatsby CLI. But, now one new error comes up - https://paste.fedoraproject.org/paste/z2B7XC0uDnC-UBbA~nbVcA .

**Comment by sidntrivedi** [comment]
Closing it as tried it locally after debugging and it's not working.

**Comment by sidntrivedi** [comment]
@ryzokuken @saranshbarua But, I think that it will make a good issue for the blog.

**Comment by ryzokuken** [comment]
@sidntrivedi012 do you want to work on it?

**Comment by sidntrivedi** [comment]
@ryzokuken Yes, will try to implement it.

**Comment by sidntrivedi** [comment]
screenshot from 2018-10-29 19-11-51
@ryzokuken Please take a look. Just had to add the favicon.ico file in the static folder.

**Review by ryzokuken — APPROVED** [review]
Looks good to me. @saranshbarua wdyt.

**Comment by sidntrivedi** [comment]
@ryzokuken @saranshbarua Made relevant changes required in #12 here. Please review.

**Review by ryzokuken — CHANGES_REQUESTED** [review]
This seems to be having the exact same whitespace issues. Please fix that.

Probably something wrong with your editor? Don't tell me you're using emacs.

**Comment by sidntrivedi** [comment]
@ryzokuken Please check. 

Not using Emacs this time but Sublime Text ;)

File: src/components/Bio.js [hunk 1]
```diff
@@ -27,12 +27,10 @@ class Bio extends React.Component {
...
           Written by the folks at the{' '}
-          <strong>Open Source Developers' Club</strong> who live in and around
-          JIIT, Noida, India.
-        </p>
-      </div>
-    )
-  }
-}
-
+          <strong>Open Source Developers Community</strong> who live in and around JIIT, Noida, India.
```
Comment (ryzokuken) [inline src/components/Bio.js]: Thanks to sublime, the whitespace issues seem to be fixed now. Please reindent these properly and we'd be good to go.

**Review by ryzokuken — CHANGES_REQUESTED** [review]

**Comment by sidntrivedi** [comment]
@ryzokuken Fixed the indentation. PTAL.

**Review by ryzokuken — APPROVED** [review]
LGTM.

**Pull request merged**